posthog/client.py

```diff
@@ -129,21 +131,29 @@ def identify(self, distinct_id=None, properties=None, context=None, timestamp=No
 
         return self._enqueue(msg)
 
-    def get_feature_variants(self, distinct_id, groups=None, person_properties=None, group_properties=None):
-        resp_data = self.get_decide(distinct_id, groups, person_properties, group_properties)
+    def get_feature_variants(
+        self, distinct_id, groups=None, person_properties=None, group_properties=None, geoip_disable=True
+    ):
+        resp_data = self.get_decide(distinct_id, groups, person_properties, group_properties, geoip_disable)
         return resp_data["featureFlags"]
 
-    def _get_active_feature_variants(self, distinct_id, groups=None, person_properties=None, group_properties=None):
-        feature_variants = self.get_feature_variants(distinct_id, groups, person_properties, group_properties)
+    def _get_active_feature_variants(
```

Comment:
this one should not default to true when the SDK level setting is false, which it does right now because we didn't add geoip_disable as a parameter to the call site.

I think it might be easier to work with these here if you default to `False` everywhere, explicitly input true when SDK level setting is set, and then test that this message is added to all capture/flag calls. Same for `get_feature_variants` and `get_feature_payloads` and `get_decide`